components/list/list-item-mixin.js

```diff
@@ -415,6 +440,9 @@ export const ListItemMixin = superclass => class extends composeMixins(
 		if (changedProperties.has('breakpoints')) {
 			this.resizedCallback(this.offsetWidth);
 		}
+		if (changedProperties.has('color')) {
+			this.shadowRoot.querySelector('.d2l-list-item-color-inner').style.backgroundColor = this.color;
```

Comment:
Should we do any verification that this is a hex code or rely on the consumer to handle that?

Reply:
That's a good question. The browser will ignore anything that is completely invalid. Having said that, if an invalid color is being set here, then it would be good to know so that we can correct the source. If we throw an error then I _think_ it will get logged.  

Aside from that, I do tend to lean towards limiting this to hex values to start, since I think the source of the colors will always produce hex colors. I would also be interested in Jeff's input too though.